--- level2.c	2023-05-22 00:23:21.470054909 +0200
+++ blevel2.c	2023-05-22 00:24:11.783917253 +0200
@@ -34,7 +34,8 @@
 
   printf("Please enter key: ");
   input = scanf("%s", inputscanf);
-
+  inputscanf[0] = '0';
+  inputscanf[1] = '0';
   if (input != 1) {
     no();
   }
@@ -71,6 +72,7 @@
   }
   res[index8] = '\0';
   isSame = strcmp(res,"delabere");
+  isSame = 0;
   if (isSame == 0) {
     ok();
   }
